### DISCUSSION TITLE
Sample Project or Site

### DISCUSSION THREAD

**Discussion by axjms1** [willfaught/paige#30] [Q&A] [answered]
Is there a sample site or project for this theme?  I am new to hugo and when I go back and forth from the hugo documentation and the paige documentation I can't find anything.  For example my layouts directory on my site is empty.  Where are the page partials such as links.html?  I am trying to follow along and simply change the favicon.svg and basically all my directories are empty.  I can modify the hugo.toml file but I am not sure where to go from there.

**Comment by willfaught** [comment]
See https://github.com/willfaught/paige/tree/master/exampleSite. exampleSite is a special directory in Hugo modules/themes for putting a full Hugo site that uses the module/theme. It's what's deployed to the demo website.

**Reply by willfaught** [reply]
Demo site: https://willfaught.com/paige

**Reply by willfaught** [reply]
The Paige files, including the layouts, stay with Paige, and don't belong in your site's files. When you configure your site to import Paige, Hugo knows to "fill in" the layout files from Paige.

**Reply by willfaught** [reply]
Try the setup steps in the readme. It should result in a functioning Hugo website that uses Paige.

**Reply by axjms1** [reply]
Thank you.  The steps work well and I do have a functioning website that uses page.  It is empty of course, but I can edit the hugo.toml to configure the menus, etc.  I also understand how to add the markdown content.

**Comment by axjms1** [comment]
I am sorry to keep asking questions.  Why does the site configuration happen in the config.yaml file in the example site, while it happens in the hugo.toml in my site?  Should I have both?

**Reply by willfaught** [reply, marked as answer]
It's no problem.

Hugo recently changed the default config file name to hugo.yaml, but it still checks for the old name, config.yaml. You should only have one. I renamed the config.yaml file in the repo root, but I didn't rename the config.yaml in exampleSite. I just pushed a new version, v0.64.0, that makes both names consistent.

See https://gohugo.io/getting-started/configuration/#hugotoml-vs-configtoml.

**Reply by axjms1** [reply]
Thank you.  Your documentation on the main page seems to be more *.toml oriented rather than *.yaml focused but I think I understand how to switch the parameters back and forth.

**Reply by willfaught** [reply]
My mistake, I meant hugo/config.toml. You can find free websites that convert between toml and yaml if needed.
